can/interfaces/pcan/pcan.py

```diff
@@ -555,6 +556,10 @@ def _recv_internal(
             elif result & (PCAN_ERROR_BUSLIGHT | PCAN_ERROR_BUSHEAVY):
                 log.warning(self._get_formatted_error(result))
 
+            elif result == PCAN_ERROR_ILLDATA:
+                # Ignore this
```

Comment:
Add some explanation here and fix the import order, then i'll merge.

Reply:
Done. Thanks.